CHANGELOG-3.7.md

```diff
@@ -85,15 +87,21 @@ particular aspect of Ice.
   - [Ruby Changes](#ruby-changes-3)
 - [Changes in Ice 3.7.0](#changes-in-ice-370)
   - [General Changes](#general-changes-6)
-  - [C++ Changes](#c-changes-17)
-  - [C# Changes](#c-changes-18)
+  - [C++ Changes](#c-changes-18)
+  - [C# Changes](#c-changes-19)
   - [Java Changes](#java-changes-7)
   - [JavaScript Changes](#javascript-changes-7)
   - [Objective-C Changes](#objective-c-changes-2)
   - [PHP Changes](#php-changes-5)
   - [Python Changes](#python-changes-8)
   - [Ruby Changes](#ruby-changes-4)
 
+# Changes since Ice 3.7.10
+
+## C++ Changes
+
+- Fix bug with the Ice.ServerIdleTime affecting Apple platforms. (https://github.com/zeroc-ice/ice/issues/2025)
```

Comment:
We should consider creating a separate issue to clarify the Apple bug, alternatively, add a comment to the existing issue. It's not evident what the bug was unless you look at the code changes.